scripts/offices.py

```diff
@@ -109,7 +109,7 @@ def door_label(members, logo=True):
             with Image(filename=LOGO) as logo:
                 logo.transform(resize=f'{WIDTH}x{HEIGHT}')
                 draw.composite('over', 200, 0, logo.width, logo.height, logo)
-        draw.font_size = 80 if len(members) >= 4 else 90
+        draw.font_size = 70 if len(members) > 4 else (80 if len(members) == 4 else 90)
```

Comment:
This start to be harder to read. I suggest the following:
```suggestion
        if len(members) > 4:
            draw.font_size = 70
        elif len(members) == 4:
            draw.font_size = 80
        else:
            draw.font_size = 90
```

Reply:
Ok.